src/components/missions/Missions.js

```diff
@@ -0,0 +1,95 @@
+import './missions.css';
+import { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchMissions, changeStatus } from '../../redux/missions/missions';
+import Details from './Details';
+
+const Missions = () => {
+  const [openModal, setOpenModal] = useState(false);
+  const [modalMission, setModalMission] = useState({
+    name: '',
+    description: '',
+  });
+  const missions = useSelector((state) => state.missions);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(fetchMissions);
+    // eslint-disable-next-line
+  }, []);
+
+  const toggleStatus = (id) => {
+    dispatch(changeStatus(id));
+  };
+
+  const showModal = (name, description) => {
+    setOpenModal(true);
+    setModalMission({
+      name,
+      description,
+    });
+  };
+
+  return (
+    <div id="missions-section" style={{ overflow: openModal ? 'hidden' : 'scroll' }}>
+      <table>
+        <thead>
+          <tr>
+            <th>Mission</th>
+            <th className="desc-header">Description</th>
+            <th>Status</th>
+            <th> </th>
+          </tr>
+        </thead>
+        <tbody>
+          {missions.map((mission) => (
+            <tr key={mission.mission_id}>
+              <td className="mission-name">
+                {mission.mission_name}
+                <button
+                  type="button"
+                  className="see-more"
+                  onClick={() => showModal(mission.mission_name, mission.description)}
+                >
+                  See more
+                </button>
+              </td>
+
+              <td className="mission-desc">{mission.description}</td>
+
+              <td className="status">
+                {!mission.reserved && <span>NOT A MEMBER</span>}
+                {mission.reserved && (
+                  <span style={{ backgroundColor: '#18a2b8' }}>Active Member</span>
+                )}
+              </td>
+
+              <td className="join-mission">
+                <button
+                  type="button"
+                  className="join-btn"
+                  onClick={() => toggleStatus(mission.mission_id)}
+                  style={{
+                    borderColor: mission.reserved ? '#d90429' : '#343a40',
+                    color: mission.reserved ? '#d90429' : '#343a40',
+                  }}
```

Comment:
[OPTIONAL] Inline styling is one of CSS bad practices. When you add inline styles, it makes it difficult to revise the design of your site and makes your code harder to manage.
I would recommend moving the style out of the `div`. Because you are using a variable in the style, try to define a constant variable `style` in the `Missions ` component and assign it to the style attribute. Checkout the example below:
```js
function MyComponent(){
    const myComponentStyle = {
      color: 'blue',
      lineHeight: 10,
      padding: '1.5em',
    }
    return <div style={myComponentStyle}> Inline Styled Component</div>
}
```